apps/vs-code-designer/src/app/commands/generateDeploymentScripts/generateDeploymentScripts.ts

```diff
@@ -36,8 +39,24 @@ export async function generateDeploymentScripts(context: IActionContext, project
     ext.outputChannel.appendLog(localize('initScriptGen', 'Initiating script generation...'));
 
     addLocalFuncTelemetry(context);
-    const commandIdentifier = extensionCommand.generateDeploymentScripts;
-    showPreviewWarning(commandIdentifier);
+    showPreviewWarning(extensionCommand.generateDeploymentScripts);
+    const parameterizeConnectionsSetting = getGlobalSetting(parameterizeConnectionsInProjectLoadSetting);
+
+    if (!parameterizeConnectionsSetting) {
+      const message = localize(
+        'parameterizeInDeploymentScripts',
+        'Allow connections to be parameterized? Otherwise the generation of deployment scripts will be cancelled.'
```

Comment:
Does the following work?

```suggestion
        'Allow connection parameterization? Declining cancels deployment scripts generation.'
```